jsf/multi-jsf-installer/pom.xml

```diff
@@ -57,12 +57,12 @@
             <id>mojarra-2.x</id>
             <properties>
                 <jsf-impl-name>mojarra</jsf-impl-name>
-                <jsf-version>2.2.17</jsf-version>
+                <jsf-version>2.3.5.SP1</jsf-version>
             </properties>
             <dependencies>
                 <dependency>
-                    <groupId>com.sun.faces</groupId>
-                    <artifactId>jsf-api</artifactId>
+                    <groupId>org.jboss.spec.javax.faces</groupId>
+                    <artifactId>jboss-jsf-api_2.3_spec</artifactId>
```

Comment:
It looks like with this change, it will no longer be possible to install Mojarra 2.2.x versions? I think it would be better to have one profile for Mojarra 2.3.x (mojarra-2.x) and another profile for Mojarra 2.2.x (mojarra-2.2.x).